### PR TITLE
[PC-10974] Add call to action message and icon comm inapp

### PR DESCRIPTION
**Pull request by yorickeando**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10974

The Cheatcodes show an icon that is too big for the Call To ActionButton. This will be fixed in the adequate ticket: https://passculture.atlassian.net/browse/PC-11177

## Checklist

I have:

- [X] Made sure the title of my PR follows the convention `[PC-XXXXX] <summary>`.
- [X] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [X] Written **unit tests** native (and web when implementation is different) for my feature.
- [X] Added a **screenshot** for UI tickets.
- [X] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" : `https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a`

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|   Capture d’écran 2021-10-12 à 10 48 30 1  |         |                |                  |                  |

### CODE REVIEW

File: src/features/profile/utils.ts [hunk 1]
```diff
@@ -37,3 +38,12 @@ export const matchSubscriptionMessagePopOverIconToSvg = (iconName: string | unde
       return Info
   }
 }
+
+export const matchSubscriptionMessageCTAIconToSvg = (iconName: string | undefined) => {
+  switch (iconName) {
```
Comment (marionnousvalentin) [inline src/features/profile/utils.ts:43]: ❓ tu ne peux pas réutiliser  matchSubscriptionMessagePopOverIconToSvg ??? on va recoder deux fois la meme chose

Comment (yorickeando) [reply src/features/profile/utils.ts:43]: Pas la même logique métier d'affichage : 
- Dans un cas si on reçoit une string incconue, on affiche une icône par défaut
- Dans l'autre, on n'affiche rien

File: src/features/cheatcodes/pages/AppComponents/AppComponents.tsx [hunk 2]
```diff
@@ -833,6 +853,23 @@ export const AppComponents: FunctionComponent = () => {
...
           <Button title="Unknown key" onPress={() => setPopOverIconString('Blablou')} />
         </AlignedText>
+        <Text>Affichage Call To Action message</Text>
+        <AlignedText>
+          <Button
+            title="Toggle Call To Action"
+            onPress={() => {
+              return callToActionMessage
+                ? setCallToActionMessage(undefined)
+                : setCallToActionMessage('Tu peux cliquer ici')
```
Comment (marionnousvalentin) [inline src/features/cheatcodes/pages/AppComponents/AppComponents.tsx:863]: ❓ c'est vu dans les cas limite ça ? ^^et quid de la tradiuction

Comment (marionnousvalentin) [reply src/features/cheatcodes/pages/AppComponents/AppComponents.tsx:863]: oké

File: src/features/profile/utils.ts (hunk 1 above)
Comment (yorickeando) [reply src/features/profile/utils.ts]: ui

**Review by marionnousvalentin — APPROVED** [review]

**Pull request merged**